packages/@react-aria/listbox/docs/useListBox.mdx

```diff
@@ -417,6 +497,58 @@ function Option({item, state}) {
 </ListBox>
 ```
 
+## Asynchronous loading
+
+This example uses the [useAsyncList](../react-stately/useAsyncList.html) hook to handle asynchronous loading
+of data from a server. You may additionally want to display a spinner to indicate the loading
+state to the user, or support features like infinite scroll to load more data.
+
+```tsx example
+import {useAsyncList} from '@react-stately/data';
+
+function AsyncLoadingExample() {
```

Comment:
should this example stop loading pretty quickly? it's only going to raticate for me

Reply:
It doesn't infinite scroll, no. That would require additional code be added to the component to handle scrolling...